Resources/doc/index.md

```diff
@@ -141,6 +141,22 @@ be omitted from the DSNs. Also make sure to use the sentinel port number
 (26379 by default) in the DSNs, and not the default Redis port.
 You can find more information about this on [Configuring Sentinel](https://redis.io/topics/sentinel#configuring-sentinel).
 
+A setup using `RedisCluster` from `phpredis`  could look like this:
+
+``` yaml
+snc_redis:
+    clients:
+        default:
+            type: phpredis
+            alias: default
+            dsn: ["redis://localhost/0"]
```

Comment:
I saw that `RedisCluster` can accept multiple hosts that is called `seeds`. It's currently not possible and result in

```
In SncRedisExtension.php line 258:
                                                  
  [RuntimeException]                              
  Support for RedisArray is not yet implemented.  
```

This error message should be displayed only if `options.cluster` is not set. What do you think?

Reply:
I agree with your suggestion, i'll add check asap.